docs/examples/index.md

```diff
@@ -0,0 +1,38 @@
+---
+id: index
+title: Overview
+---
+We write some example codes of `neon-js` for developers to refer to. The examples are expected to cover all cases and methods on different levels.
+
+## Cases
+- Network Initiation
+  - MainNet/TestNet
+  - PrivateNet
+
+- Wallet Operation
+  - Create wallet
+  - Import wallet
+  - Export wallet
+
+- Native Asset
+  - Get Balance
+  - Send Neo and Gas
+  - Claim Gas
+
+- Smart Contract
+  - Invocation Transaction
+  - Local Invoke
```

Comment:
btw there is nothing `local` about this invocation. I would suggest changing the name.

Reply:
Changed to `Read Invoke - No Transaction Broadcast`.